packages/core/src/accordion/AccordionPanel.css

```diff
@@ -1,18 +1,20 @@
 .saltAccordionPanel {
   display: grid;
   grid-template-rows: 0fr;
-  transition: grid-template-rows var(--salt-duration-perceptible) cubic-bezier(0.4, 0, 0.2, 1);
+  transition: grid-template-rows var(--salt-duration-perceptible) ease-in-out, opacity var(--salt-duration-perceptible) ease-in-out;
+  opacity: 0;
 }
 
 .saltAccordionHeader[aria-expanded="true"] + .saltAccordionPanel {
   grid-template-rows: 1fr;
+  opacity: 1;
 }
 
 .saltAccordionPanel-inner {
   overflow: hidden;
 }
 
 .saltAccordionPanel-content {
-  padding: var(--salt-spacing-200);
   padding-left: calc(var(--salt-spacing-200) + var(--salt-size-icon));
+  padding-block: var(--salt-spacing-300);
```

Comment:
Just realize `padding-block` comes in from Chrome 87, may have trouble for min browser conv